app/models/audience.rb

```diff
@@ -0,0 +1,27 @@
+class Audience
```

Comment:
I wonder if this whole class just wants to disappear. 

It's a lot of code around an constant array of two strings. Can we put that somewhere else?